### PR TITLE
Uniswap route

### PR DESCRIPTION
**Pull request by Vaibhav20201**
- [x] Code linting
- [x] Added uniswap route to:-
    - [x] Polygon
    - [x] Arbitrum
    - [x] Optimism

### CODE REVIEW

**Review by KaymasJain — APPROVED** [review]

File: hardhat.config.ts [hunk 1]
```diff
@@ -122,6 +122,12 @@ const config: HardhatUserConfig = {
       chainId: 137,
       accounts: [`0x${process.env.PRIVATE_KEY}`],
       gasPrice: 70000000000,
+    },
+    optimism_mainnet: {
+      url: `https://eth-mainnet.alchemyapi.io/v2/${alchemyApiKey}`,
```
Comment (Vaibhav20201) [inline hardhat.config.ts:127]: need to add the alchemy key of optimism here

Comment (Vaibhav20201) [reply hardhat.config.ts:127]:
```suggestion
      url: `https://opt-mainnet.g.alchemy.com/v2/${alchemyApiKey}`,
```

**Comment by piyush2-0** [comment]
Newly Deployed Addresses:
Polygon new Implementation: 0x84E6b05A089d5677A702cF61dc14335b4bE5b282
Polygon resolver: 0xC6D3Cf6cC962DC00C288cea4ba0DbE2e76869B1B
Arbitrum new Implementation: 0xDAa3F68f0033d8ad252e0a53b402943221705714
Arbitrum resolver: 0x84E6b05A089d5677A702cF61dc14335b4bE5b282
Optimism Aggregator: 0xDAa3F68f0033d8ad252e0a53b402943221705714
Optimism Proxy: 0x84E6b05A089d5677A702cF61dc14335b4bE5b282
Optimism Resolver: 0x12D3E615C755b6a8B15330263b2137eED97f80d9

**Comment by Vaibhav20201** [comment]
Updated resolver address:-
Optimism - [here](https://optimistic.etherscan.io/address/0xD1aF3EE5D22025c2070c10bB826d5Efb9aAE1c53#code)

**Review by KaymasJain — APPROVED** [review]

**Comment by piyush2-0** [comment]
Updated resolver addresses:
Optimism: [here](https://optimistic.etherscan.io/address/0x33d8f735dd64cec51d212616bca5ad9b7769cd34)
Polygon: [here](https://polygonscan.com/address/0xB7B4bB4C510a0881542d283C71F292AbdD071fF3)
Arbitrum: [here](https://arbiscan.io/address/0x12D3E615C755b6a8B15330263b2137eED97f80d9)

**Comment by Vaibhav20201** [comment]
Polygon implementation update - [txnHash](https://polygonscan.com/tx/0xc1859489373b9de827e3befd94696151254f5304e38ae29a4d027b06cd24900d)
Arbitrum implementation update - [txnHash](https://arbiscan.io/tx/0x967a44c4e1fd58490d626e75a355c6c80ae6adf2f42c09974945375e1a1ec4ab)

**Pull request merged**